src/modules/endpoint/sagas/watchFetchNews.js

```diff
@@ -71,18 +76,15 @@ export function * fetchNews (
       }
       yield put(insert)
     } else {
-      const allAvailableLanguages =
-        path === null
-          ? new Map(languages.map(language => [language.code, null]))
-          : null
+      const allAvailableLanguages = newsId ? new Map(languages.map(language => [language.code, null])) : null
```

Comment:
Why do we do that?

Reply:
From my understanding of the code. This is to prevent having language choices when a news item is clicked. As news items will be in one language.